### DISCUSSION TITLE
Tilt Map View

### DISCUSSION THREAD

**Discussion by ghost** [react-native-maps/react-native-maps#4692] [Q&A]
### Summary

I am having an issue in react-native-maps version 1.4.0, I want to tilt the mapView for perspective view, Is there any solution available then please provide or else please add animateToViewingAngle and animateToBearing back in the new PR. 

### Reproducible sample code

```TSX
<MapView
          ref={mapRef}
          onLayout={() => {
            mapRef.current?.animateToBearing(125);
            mapRef.current?.animateToViewingAngle(angle);
          }}
          onRegionChange={() => {
            setAngle(45);
          }}
          zoomControlEnabled={true}
          userLocationCalloutEnabled={true}
          // mapPadding={{ top: 0, right: 0, bottom: 800, left: 0 }}
          zoomEnabled={true}
          minZoomLevel={5}
          maxZoomLevel={20}
          showsMyLocationButton={true}
          provider={PROVIDER_GOOGLE} // remove if not using Google Maps
          style={styles.map}
          initialRegion={{
            latitude: coords.latitude,
            longitude: coords.longitude,
            latitudeDelta: 1 / 300,
            longitudeDelta: 2 / 300,
          }}
        >
          <Marker
            coordinate={{
              latitude: coords.latitude,
              longitude: coords.longitude,
            }}
            pinColor="#0C8CE1"
          >
            <FeatherIcon size={35} name="navigate" color="#0C8CE1" />
          </Marker>
          {renderUsersLocation()}
        </MapView>
```

### Steps to reproduce

I am having an issue in react-native-maps version 1.4.0, I want to tilt the mapView for perspective view, Is there any solution available then please provide or else please add animateToViewingAngle and animateToBearing back in the new PR. 

### Expected result

I want to tilt the mapView like google maps on tilt by multi-fingers

### Actual result

At present there is no option available to tilt 

![Tilted map](https://user-images.githubusercontent.com/112846374/225002897-4aa80d75-7f13-4c7f-ae9f-80394caddc05.png)

### React Native Maps Version

1.4.0

### What platforms are you seeing the problem on?

Android

### React Native Version

0.71.4

### What version of Expo are you using?

Not using Expo

### Device(s)

android 12

### Additional information

Please drop a e-mail regarding any solution on aakash.rathore@inventam.com
or else aakash.rathore321@gmail.com

**Comment by vitorcont** [comment]
No news about this topic?

**Comment by adeel39** [comment]
Do you have any updates on this?
